base/mpfr.jl

```diff
@@ -588,56 +582,54 @@ for f in (:log, :log2, :log10)
                               "with a complex argument. Try ", $f, "(complex(x)).")))
         end
         z = BigFloat()
-        ccall(($(string(:mpfr_,f)), :libmpfr), Int32, (Ptr{BigFloat}, Ptr{BigFloat}, Int32), &z, &x, ROUNDING_MODE[])
+        ccall(($(string(:mpfr_,f)), :libmpfr), Int32, (Ref{BigFloat}, Ref{BigFloat}, Int32), z, x, ROUNDING_MODE[])
         return z
     end
 end
 
 function log1p(x::BigFloat)
     if x < -1
         throw(DomainError(x, string("log1p will only return a complex result if called ",
-                                    "with a complex argument. Try log1p(complex(x)).")))
+                          "with a complex argument. Try log1p(complex(x)).")))
```

Comment:
I had it before as you suggested, but @KristofferC said it looked strange so I changed it. I personally don't care either way and don't think it's a big deal, but will change according to a consensus..